### PR TITLE
Fix #173 (Change license to 0BSD)

### LINKED ISSUE #173: thoughts about a more permissive license
Hi there,

Thanks for making a very useful library. We're wondering if you've been giving thoughts about using a different library like BSD or MIT which makes it easier for companies to integrate software. There is also the [GPL linking exception](https://en.wikipedia.org/wiki/GPL_linking_exception) which could help as well.

Thanks !
- Benjamin

### PR DESCRIPTION
**Pull request by danmar**

### CODE REVIEW

**Comment by danmar** [comment]
@firewave @amai2012 @rikardfalkeborn @PKEuS @keinflue @chrchr-github @HolgiHo @orbitcowboy @Dmitry-Me @dummyunit @patrickdowling @jannick0 @IOBYTE @katahiromz @larslj @JIghtuse @JohnSiegel @mptre @diamante0018 @bavison @al42and @josecm @Ken-Patrick @versat @o-bolshakov @mitsujin @diorcety @rsBNT @matthiaskrgr @x29a @uburuntu @mathbunnyru @zingsheim @svenpanne

I would like to change the license for simplecpp to something that will be easier to use commercially.

Basically my philosophy is that companies should be allowed to use and modify simplecpp source code. My philosophy is that no attribution is required; they can use simplecpp without saying it to anybody.

Please feel free to comment if you have some objections.

**Comment by diamante0018** [comment]
Feel free to change the licence.
Good luck with the future of this project

**Comment by larslj** [comment]
My contribution was very minor. It's fine by me to change the license.

**Comment by matthiaskrgr** [comment]
Looks good to me, no objections on my side :+1:

**Comment by katahiromz** [comment]
No objection from me.

**Comment by orbitcowboy** [comment]
Looks good to me as well. Keep up your great work!

**Comment by mathbunnyru** [comment]
LGTM.

**Review by versat — APPROVED** [review]

**Comment by Ken-Patrick** [comment]
lgtm

Le 16 décembre 2023 11:33:29 GMT+01:00, "Daniel Marjamäki" ***@***.***> a écrit :

**Comment by josecm** [comment]
Ok by me. Keep up the great work @danmar !

**Comment by HolgiHo** [comment]
LGTM. Thank you for all your efforts!

**Comment by o-bolshakov** [comment]
Hi Daniel,

BSD zero clause looks like a good choice for your purpose. Thanks for your work!

Respectfully, Oleg

**Comment by rikardfalkeborn** [comment]
Lgtm

**Review by chrchr-github — APPROVED** [review]

**Comment by amai2012** [comment]
No objection from me.

**Comment by firewave** [comment]
LGTM

**Comment by JIghtuse** [comment]
LGTM. Good luck with simplecpp and cppcheck!

**Review by x29a — APPROVED** [review]

**Comment by diorcety** [comment]
LGTM

**Review by firewave — APPROVED** [review]

**Review by larslj — APPROVED** [review]

**Comment by bavison** [comment]
Very happy with the proposal, and thanks for asking even those of us who have only played a minor part!

**Review by JIghtuse — APPROVED** [review]

**Review by josecm — APPROVED** [review]

**Comment by danmar** [comment]
ok I will merge this now.. and release a simplecpp with this license soon..

**Comment by zingsheim** [comment]
OK for me, too.

 Sorry for the late reply
 Frank

Am Samstag, dem 16.12.2023 um 02:33 -0800 schrieb Daniel Marjamäki:
